src/cli/build_command.zig

```diff
@@ -35,6 +35,26 @@ const BundleV2 = @import("../bundler/bundle_v2.zig").BundleV2;
 var estimated_input_lines_of_code_: usize = undefined;
 
 pub const BuildCommand = struct {
+    const compile_define_keys = &.{
+        "process.platform",
+        "process.arch",
+    };
+    const compile_define_values = &.{
+        switch (Environment.os) {
+            .linux => "'linux'",
+            .mac => "'darwin'",
+            .windows => "'win32'",
+            else => "unknown",
+        },
+        switch (Environment.isX64) {
```

Comment:
this switch looks weird. i would've just used `switch (builtin.target.cpu.arch)`

make the else a compile error